source/_integrations/knx.markdown

```diff
@@ -121,6 +115,35 @@ Connection parameters are set up when adding the integration and can be changed
 
 Use `route back` if your tunneling server is located on a different network.
 
+## KNX Secure
+
+The KNX integration currently supports IP secure tunneling.
+IP secure via routing and data secure are currently not supported.
+
+In order to use IP Secure you will have to chose "Tunneling" -> "TCP with IP Secure" in the config flow.
+
+You can configure the IP Secure credentials either manually or by providing a `.knxkeys` file, which you can obtain by exporting the keyring in ETS as seen in the screenshot below.
+
+<p class='img'>
+  <img src='/images/integrations/knx/export_keyring_ets.png' alt='Screenshot: Export Keyring in ETS5'>
+</p>
+
+The `.knxkeys` file has to be placed in `config/.storage/knx/yourfile.knxkeys`.
+
+If you decide to configure IP Secure manually you will need the user ID (starts with 2), the user password and the device authentication password.
```

Comment:
"starts with 2" = does that mean the digit "2" followed by some more digits or characters or is the first user ID the number 2, the next one would be 3?

Reply:
Hehe, was about to adjust this :). You can define 2 and 2 is the default. It depends on the tunneling channel that you would like to use though.